addons/website/models/website.py

```diff
@@ -998,20 +995,14 @@ def _get_current_website_id(self, domain_name, country_id, fallback=True):
         fallback to the first found website (no matter its `domain`) or return
         False depending on the `fallback` parameter.
 
-        If there are multiple websites for the same `domain_name`, we need to
-        filter them out by country. We return the first found website matching
-        the given `country_id`. If no found website matching `domain_name`
-        corresponds to the given `country_id`, the first found website for
-        `domain_name` will be returned (no matter its country).
+        If there are multiple websites for the same `domain_name`, the first
+        found website for `domain_name` will be returned.
```

Comment:
Is not it supposed to be impossible thanks to your SQL constraint?

Reply:
Indeed, I added that constraint later in the PR and forgot about this :grimacing: 
Thanks